### ISSUE TITLE
Path Issue with new reflex update

### ISSUE THREAD

**Issue by Ernelene** [kroo/reflex-clerk#9]
I am getting this error with the new reflex v0.5.7 any help would be appreciated.
```
[Reflex Backend Exception]
 Traceback (most recent call last):
  File "...site-packages/reflex/app.py", line 1274, in process
    if app._process_background(state, event) is not None:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "...site-packages/reflex/app.py", line 1097, in _process_background
    substate, handler = state._get_event_handler(event)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "...site-packages/reflex/state.py", line 1366, in _get_event_handler
    substate = self.get_substate(path)
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "...site-packages/reflex/state.py", line 1172, in get_substate
    raise ValueError(f"Invalid path: {path}")
ValueError: Invalid path: ['state', 'clerk_state']
```

**Comment by dentro-innovation** [comment]
yes I run into the same issue

**Comment by kroo** [comment]
Thanks @dentro-innovation @Ernelene, this issue is now addressed in [v1.0.3](https://github.com/kroo/reflex-clerk/releases/tag/v1.0.3)
